### ISSUE TITLE
Wrong Type for FileField based fields in POST type bodies

### ISSUE THREAD

**Issue by autoantwort** [tfranzel/drf-spectacular#1251]
**Describe the bug**
If I have a model with a models.FileField field, the type of this field is 'url' since the GET Request would include a url to the image, which is right. But when doing post requests, you don't have to provide an url but an file. 

**To Reproduce**
It would be most helpful to provide a small snippet to see how the bug was provoked.
```python
class Bar(models.Model):
    Image = models.FileField(upload_to="events/", null=True, blank=True)

... Boilerplate with serializers.HyperlinkedModelSerializer Router
```

The generated schema then contains 
```yml
post:
      operationId: bars_create
      tags:
      - bars
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Bar'
```
and 
`#/components/schemas/Bar` says:
```yml
Bar:
      type: object
      properties:
        image:
          type: string
          format: uri
          nullable: true
```
**Expected behavior**
If i understand https://swagger.io/docs/specification/describing-request-body/file-upload/ correctly it should be `format: binary`

**Comment by tfranzel** [comment]
This is a issue explained in the docs. Have a look at: https://drf-spectacular.readthedocs.io/en/latest/faq.html#filefield-imagefield-is-not-handled-properly-in-the-schema

tldr: the setting `COMPONENT_SPLIT_REQUEST = True` will create two instead of one component, which will then better match the behavior.

dupe of: https://github.com/tfranzel/drf-spectacular/issues/761#issuecomment-1167227490

**Comment by autoantwort** [comment]
Thank you for the hint! :) I will close this then. 

Very nice project btw! ❤️
